File: Java/Car.java
package Java;

public class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passengers;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        if (passengers != null) {
            System.out.println(
                    "The License is: " + license + " your Driver is: " + driver.name + " and number Passengers: "
                            + passengers);
        }
    }

    public Integer getPassenger() {
        return passengers;
    }

    public void setPassenger(Integer passenger) {
        if (passenger == 4) {
            this.passengers = passenger;
        } else {
            System.out.println("Numero de pasajeros no validos");
        }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    
}

File: PHP/Car.php
<?php
require_once('Account.php');
    class Car{
        $id = int;
        $license = string;
        $driver = string;
        protected $passengers = int;

        public fucntion __construct($license, $driver){
            $this->license = $license;
            $this->driver = $driver
        }

        public function printDataCar(){
            echo "license: $this->license, conductor: {$this->driver->name}, document: {$this->driver->document}";
        }

        public function printDataCar() {
        echo "Licencia: $this->license, Driver: {$this->driver->name}, Número de pasajeros: $this->passenger";
        }

        public function getPassenger() {
        return $this->passengers;
        }
        public function setPassenger($passengers) {
        
        if ($passengers == 4) {
        $this->passengers = $passengers;
        }
        else {
        echo "Necesitas asignar 4 pasajeros";
        }

        }
    }
?>